src/content/docs/apm/apm-ui-pages/events/change-tracking/deployments-page-view-impact-your-app-users.mdx

```diff
@@ -0,0 +1,189 @@
+
+You can adjust the before/after period used in these calculations and displayed on time series charts via the timepicker at the top right corner of the deployment details page.
+    <Callout variant="tip">
+      Pay close attention to the selected before/after time period as you analyze the impact of a change. If the period is too short or too long for your specific needs, just adjust it. Note that the UI will not calculate net impacts if elapsed time since the deployment is less than your selected before/after period because doing so results in skewed and generally unhelpful results.
+    </Callout>
+* Faceted findings from a variety of New Relic products including errors inbox, log monitoring, AIOps (issues, incidents, and anomalies), and more show how this deployment has affected or relates to crucial troubleshooting and analysis records. The insights featured here and how data is filtered will vary by entity type. Hover over titles of page sections to learn more about how we surface meaningful insights here.
+
+    <img
+      title="A screenshot showing some faceted findings since your deployment"
+      alt="A screenshot showing some faceted findings since your deployment"
+      src={trackingFacetedFindings}
+    />
+    <Callout variant="tip">
+      The page-level timepicker also affects facet insights from other New Relic products. Try zeroing in on the time period after a deployment that matters most to you.
+    </Callout>
+* A link (**See all deployments on this entity**) to a page containing the aforementioned deployments list page filtered to this entity. Use this to quickly zoom out and see all recent changes on this part of your system.
+* Links to entities related to this deployment's entity. This helps you move across your stack as you look for the source of a problem or measure the impact of changes stemming from this deployment.
+* All the data supplied when the deployment was recorded, which helps anyone quickly understand context—regardless of whether they made the change or not.
+    <Callout variant="tip">
+      When you record a deployment, try submitting a URL for the changelog attribute. The UI will render it as a clickable link that serves as a convenient jumping-off point to your source repo management system.
+    </Callout>
+
+### Change the before/after comparison period [#change-window]
+
+The **Deployment details** page hinges on the idea that records and signals generated over a period of time leading up to this deployment are being compared with a period of equal length following this deployment. You can change the length of that period using a time picker in the top-right corner. Note that this will affect both time series charts and other UI elements featuring data-driven insights.
+
+<img
```

Comment:
Although I see this screenshot (tracking_screenshot-crop_comparison-window.png) and the following one (tracking_screenshot-crop_compare-deployments.png) are different and the src variable is right, when I load the build I see the same screenshot. I don't know why....

Reply:
@nbaenam I am seeing two separate images in the images directory, the import statements and in the <img> tags. Maybe this is a cache issue? I'll check with our developers.